.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-object-rest-spread"]
+  "presets": ["cf"],
```

Comment:
https://github.com/cloudflare/cf-ui/issues/22

We can't even use cf preset in our dashboard. It's a very strange collection of extremely experimental constructs and a very constraining, but not spec-conforming subset of ES2015. It's essentially giving us a different language that's neither a superset nor subset of ES2015. It's too complicated. Also, we want to use ES6 modules and Webpack at some point so consumers of cf-ui can tree shake the generated library code.